_posts/2020-05-27-tls-announcement.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Announcement: API endpoint update to TLS v1.2"
+slug: "announcement-api-endpoint-update-to-tls-v1.2"
+category: blog
+post_image: /assets/images/tls_update.png
+author: kevin
+date: 2020-05-25 09:00:00
+excerpt: "OnePageCRM will be removing support for TLS1.0 and TLS 1.1 on the 1st July. Please review as you will be required to update if you are using the OnePageCRM API."
+graphic: /assets/images/tls_update.png
+---
+
+
+Security is very important to us here at OnePageCRM, so we must continuously update every aspect of our software stack to ensure that our applications and your data remains safe and secure.
+
+When you visit any secure https site, including OnePageCRM, your connection is encrypted. The setup of this encryption happens during an initial handshake between your browser and our servers. This encryption is called Transport Layer Security (TLS) or, formally, Secure Sockets Layer (SSL).
+
+There are many versions of SSL/TLS.
+
+* SSL 2.0 => Depreciated in 2011.
+* SSL 3.0 => Depreciated in 2015.
+* TLS 1.0 => Depreciated in 2020.
+* TLS 1.1 => Depreciated in 2020.
```

Comment:
@kevGalway I think here we have a typo: should be `deprecated` instead of `depreciated`, because it didn't loose it's price, but became obsolete. All lines above

Reply:
Good catch. Updated